pkg/middleware/middleware.go

```diff
@@ -33,7 +33,8 @@ func AddDefaultResponseHeaders(cfg *setting.Cfg) macaron.Handler {
 				return
 			}
 
-			if !strings.HasPrefix(c.Req.URL.Path, "/api/datasources/proxy/") {
+			if !strings.HasPrefix(c.Req.URL.Path, "/api/datasources/proxy/") ||
+				strings.HasPrefix(c.Req.URL.Path, "/public/plugins/") {
```

Comment:
Think this should be "AND NOT" rather than "OR"

```suggestion
			if !strings.HasPrefix(c.Req.URL.Path, "/api/datasources/proxy/") &&
				!strings.HasPrefix(c.Req.URL.Path, "/public/plugins/") {
```

Reply:
Based on the other discussions we shouldn't need to exclude plugin assets here

```suggestion
			if !strings.HasPrefix(c.Req.URL.Path, "/api/datasources/proxy/") {
```